addon/components/sl-textarea.js

```diff
@@ -104,11 +92,11 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, ComponentInpu
     /**
      * The `spellcheck` HTML attribute value
      *
-     * Accepted values are true, false, "default" (default), "true", or "false".
+     * Accepted values "false" (default) or "true".
      *
-     * @type {Spellcheck}
+     * @type {Boolean}
      */
-    spellcheck: Spellcheck.Direction,
+    spellcheck: false,
```

Comment:
Have you thoroughly tested this with bound values in an app scenario?  Boolean `false` is different than a string `"false` (and likewise for `true`/ `"true"`).  Are we sure we're getting the desired behavior out of this configuraiton?